scripts/install.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-ERA_TEST_NODE_REPO="https://github.com/matter-labs/era-test-node"
+ANVIL_ZKSYNC_REPO="https://github.com/matter-labs/anvil-zksync"
 
 function script_usage() {
     cat << EOF
```

Comment:
There are a couple of lines that mention "Era Test Node" just below this (damn you Github for not allowing to comment on arbitrary lines)